group17-s1/a2/a.txt

```diff
@@ -0,0 +1,23 @@
+Q: If only Alice has some type of secret/private key, can she use it to give Bob her secret message?
+A: In case of AES, no. Both parties have to have the same secret key in order to en- and decrypt messages.
+   In case of RSA, also no. While Alice can use her private key to encrypt a message, it can be decrypted by ecveryone, including Eve
```

Comment:
```suggestion
   In case of RSA, also no. While Alice can use her private key to encrypt a message, it can be decrypted by everyone, including Eve
```